starters/serverless-apollo-contentful/jest.config.ts

```diff
@@ -7,7 +7,12 @@ const config: JestConfigWithTsJest = {
 	collectCoverageFrom: ['src/**/*.ts'],
 	coverageReporters: ['html', 'json', 'lcov', 'text', 'clover'],
 	coverageDirectory: 'coverage',
-	coveragePathIgnorePatterns: ['/node_modules/', '/test/', '/mocks/'],
+	coveragePathIgnorePatterns: [
+		'/node_modules/',
+		'/test/',
+		'/mocks/',
+		'/index.ts',
```

Comment:
Does this include index files not in the root folder?

e.g., `/some/path/index.ts`?

Reply:
It does ignore

Reply:
It does. Here is the explanation: https://stackoverflow.com/a/64979756/6523474
Here is after: 
![image](https://user-images.githubusercontent.com/15368874/218734140-7a09ede2-fe49-4f7e-b8e2-03e23a62c364.png)

Here before: 

![image](https://user-images.githubusercontent.com/15368874/218734305-89198461-b56d-483f-ab30-60d7654ce5d6.png)